common/libs/nodes/rpc/client.js

```diff
@@ -0,0 +1,69 @@
+// @flow
+import { randomBytes } from 'crypto';
+import { encodeData } from './utils';
+import type {
+  RPCRequest,
+  JsonRpcResponse,
+  CallRequest,
+  GetBalanceRequest,
+  EstimateGasRequest
+} from './types';
+
+// FIXME is it safe to generat that much entropy?
+function id() {
```

Comment:
v3 sends random ids with every request, no idea if it is because of cache or it was required some time ago